Issue archiving beta (#6980)
getsentry/sentry-docs: src/docs/product/issues/states-triage/index.mdx

---
title: Issue Status
sidebar_order: 30

> description: "Learn about issue status and how to triage issues."

Author: ```suggestion
description: "Learn about Issue status and how to triage Issues."
```

> description: "Learn about Issue status and how to triage Issues."

Author: ```suggestion
description: "Learn how Issue status works and and how to triage Issues."
```

---

> You can sort and prioritize Issues on the Issues page in Sentry.io by paying attention to the status tags attached to every Issue. It's important to note that an Issue can only have one status at a time.

Author: ```suggestion
You can sort and prioritize Issues on the [**Issues** page](https://sentry.sentry.io/issues/) in Sentry by paying attention to the status tags attached to every Issue. It's important to note that an Issue can only have **one status at a time.**
```

> You can sort and prioritize Issues on the [**Issues** page](https://sentry.sentry.io/issues/) in Sentry by paying attention to the status tags attached to every Issue. It's important to note that an Issue can only have **one status at a time.**

Author: ```suggestion
Paying attention to to the status tags attached to Issues on the [**Issues** page](https://sentry.sentry.io/issues/) in Sentry, can help you triage and prioritize problems with your application that are important to you. Keep in mind that an Issue can only have **one status at a time.**
```

Author: @robinrendle what do you think of that ^^ ?

Here's a list of all the statuses and the conditions for creating them:

> | Status         | Condition                                             | Custom Search Term |

Reviewer: Can we just call this `Search Term`? "Custom" here feels unnecessary.

> | Status     | Condition                                             | Custom Search Term |
> | ---------- | ----------------------------------------------------- | ------------------ |
> | New        | An issue that was created in the last 3 days.         | is:new             |
> | Ongoing    | An issue that was created more than 3 days ago.       | is:ongoing         |
> | Escalating | An issue that's exceeded its forecasted event volume. | is:escalating      |
> | Regressed  | A resolved issue that's come up again.                | is:regressed       |
> | Archived   | An issue that's been marked as archived.              | is:archived        |
> | Resolved   | An issue that's been marked as fixed.                 | is:resolved        |

Author: ```suggestion
| Status     | Condition                                             | Custom Search Term |
| ---------- | ----------------------------------------------------- | ------------------ |
| **New**        | An issue that was created in the last 3 days.         | is:new             |
| **Ongoing**    | An issue that was created more than 3 days ago.       | is:ongoing         |
| **Escalating** | An issue that's exceeded its forecasted event volume. | is:escalating      |
| **Regressed**  | A resolved issue that's come up again.                | is:regressed       |
| **Archived**   | An issue that's been marked as archived.              | is:archived        |
| **Resolved**   | An issue that's been marked as fixed.                 | is:resolved        |
```

> One way to sort the Issues that you see is by selecting a tab at the top of the page. On the “Unresolved tab” you'll find: New, Ongoing, Escalating, and Regressed issues, but you can also narrow down what you see by choosing the “For Review”, “New”, “Escalating”, “Ongoing”, or “Archived” tabs.

Author: ```suggestion
One way to limit the Issues that you see is by selecting a tab at the top of the **Issues** page. On the “Unresolved tab” you'll find: New, Ongoing, Escalating, and Regressed issues, but you can also narrow down further by choosing the “For Review”, “New”, “Escalating”, “Ongoing”, or “Archived” tabs.
```

> ![Manual vs. automatic issue status update](issue-status-docs.png)

Author: ```suggestion

```

## Manually Triaging Issues

> While some Issue statuses are added and updated automatically, you can manually Archive or Resolve an Issue, which will also change its status.

Author: ```suggestion
While some Issue statuses are added and updated automatically, you can manually `Archive` or `Resolve` an Issue, which will also change its status.

![Manual vs. automatic issue status update](issue-status-docs.png)
```

### Archive

> You can manually archive Issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the Issues list. Sentry will automatically bring an Issue back to the top of the list and change its status to `Escalating` if the events in that Issue significantly increase over a short period of time. There's also an option to archive an issue for:

Author: ```suggestion
You can manually archive Issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the Issues list. Sentry will automatically bring an Issue back to the top of the list and change its status to `Escalating` if the events in that Issue significantly increase over a short period of time. There's also an option to mark an Issue as `Archived` for:
```

- A set period of time
- Until it occurs a set number of times
- Until a set number of users are affected

### Resolve

> You can manually mark an Issue as Resolved when it’s been fixed. You can also specify further by resolving it in:

Author: ```suggestion
You can manually mark an Issue as `Resolved` when it’s been fixed. You can also specify further by resolving it in:
```

- Your next release
- A current release
- Another release

or

- By [including the issue ID in a commit](/product/releases/associate-commits/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F#resolve-issues-by-commit)

If the same Issue comes back, its status will be automatically changed to `Regressed`.

### Delete

> You can remove an Issue from the Issue list by deleting it, but it will reappear as a new Issue if it reoccurs.

Author: ```suggestion
You can remove an Issue from the Issue list by deleting it, but it will reappear as a new Issue if it recurs.
```

> There's also an option to `Delete and Discard Forever`, which will make it so that the Issue is never seen again, even if it reoccurs. Any future events will be discarded automatically and won't count towards [your quota](/product/accounts/quotas/).

Author: ```suggestion
There's also an option to `Delete and Discard Forever`, which will make it so that the Issue is never seen again, even if it recurs. Any future events tied to the permanently deleted Issue will be discarded automatically and won't count towards [your quota](/product/accounts/quotas/).
```

---
title: Issue Status
sidebar_order: 30
description: "Learn how issue status works and and how to triage issues."
---

Use the status tags attached to issues on the [**Issues** page](https://sentry.io/issues/) in Sentry to help you triage and prioritize problems with your application that are important to you. Keep in mind that an issue can only have **one status at a time.**

> Here's a list of all statuses, how they're assigned to an issue, and their custom serach term:

Reviewer 2: ```suggestion
Here's a list of all statuses, how they're assigned to an issue, and their custom search term:
```

| **New**        | An issue that was created in the last 7 days.                                                                                 | is:new             |
| **Ongoing**    | An issue that was created more than 7 days ago or has manually been marked as reviewed.                                       | is:ongoing         |
| **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). | is:escalating      |
| **Regressed**  | A resolved issue that's come up again.                                                                                        | is:regressed       |
| **Archived**   | An issue that's been marked as archived.                                                                                      | is:archived        |
| **Resolved**   | An issue that's been marked as fixed.                                                                                         | is:resolved        |

The diagram below shows how the statuses are updated automatically and manually:

![Issue Status Diagram](issue_statuses.png)

> One way to limit the issues that you see is by selecting a tab at the top of the **Issues** page. On the “Unresolved tab” you'll find: New, Ongoing, Escalating, and Regressed issues, but you can also narrow down further by choosing the “For Review”, “New”, “Escalating”, “Ongoing”, or “Archived” tabs.

Reviewer 2: Updated this for now. LMK if anything changes!

---
title: Issue Status
sidebar_order: 30
description: "Learn how issue status works and and how to triage issues."

> ---

Reviewer 2: ```suggestion
redirect_from:
  - /product/accounts/early-adopter-features/issue-archiving
---
```

Here's a list of all statuses, how they're assigned to an issue, and their custom search term:

| Status         | Condition                                                                                                                                                                                                  | Custom Search Term |
| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
| **New**        | An issue that was created in the last 7 days.                                                                                                                                                              | is:new             |
| **Ongoing**    | An issue that was created more than 7 days ago or has manually been marked as reviewed.                                                                                                                    | is:ongoing         |

> | **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). Please note that escalating issues does not work for merged/unmerged issues. | is:escalating      |

Reviewer 2: ```suggestion
| **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). Please note that escalating issues currently does not work for merged/unmerged issues. | is:escalating      |
```

> | **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). Please note that escalating issues does not work for merged/unmerged issues. We are working on fixing it [here](https://github.com/getsentry/snuba/issues/4558). | is:escalating      |

Reviewer 2: ```suggestion
| **Escalating** | An issue that's exceeded its forecasted event volume. For more details, see [Escalating Issues Algorithm](escalating-issues). Please note that escalating issues does not work for merged/unmerged issues. | is:escalating      |
```

### Resolve

You can manually mark an issue as `Resolved` when it’s been fixed. You can also specify further by resolving it in:

- Your next release
- A current release
- Another release

or

- By [including the issue ID in a commit](/product/releases/associate-commits/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F#resolve-issues-by-commit)

If the same issue comes back, its status will automatically change to `Regressed`.

### Delete

You can remove an issue from the issue list by deleting it, but it will reappear as a new issue if it recurs.

There's also an option to `Delete and Discard Forever`, which will make it so that the issue is never seen again, even if it recurs. Any future events tied to the permanently deleted issue will be discarded automatically and won't count towards [your quota](/product/accounts/quotas/).

<Note>

> You can only delete [Error Issues](/product/issues/issue-details/error-issues/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F). Other issue categories, (such as [Performance Issues](/product/issues/issue-details/performance-issues/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F)), don't support this feature.

Reviewer 2: ```suggestion
You can only delete [Error Issues](/product/issues/issue-details/error-issues/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F). Other issue categories, such as [Performance Issues](/product/issues/issue-details/performance-issues/?original_referrer=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fissues%2Fstates-triage%2F), don't support this feature.
```
